src/Application/Common/Services/IClanService.cs

```diff
@@ -229,11 +228,6 @@ await db.Entry(user)
             return new(CommonErrors.UserItemInUse(userItemId));
         }
 
-        if (user.Items.Any(i => i.ItemId == armoryItem.UserItem!.ItemId))
-        {
-            return new(CommonErrors.ItemAlreadyOwned(armoryItem.UserItem!.ItemId));
-        }
-

```

Comment:
Why do we allow borrowing an item that the player already has?

Reply:
well, the point of the PR is to remove some restrictions, I think there's nothing wrong with that.

Reply:
I don't have any arguments, I revert it

Reply:
++